app/controllers/organisation_member/base_controller.rb

```diff
@@ -1,12 +1,13 @@
 # frozen_string_literal: true
 
-class OrganisationMember::BaseController < User::BaseController
+class OrganisationMember::BaseController < ApplicationController
   # Controllers for users acting on behalf of organisations
 
   before_action :set_organisation
 
   def set_organisation
-    @organisation = Organisation.find(params[:organisation_id])
-    authorize @organisation, :act_as?
+    organisation = Organisation.find(params[:organisation_id])
+    authorize organisation, :act_as?
+    @organisation = organisation
```

Comment:
what's the difference here? Requests_controller has something similar in the same format as how it was

Reply:
Your right, this change isn't needed.